api/src/main/java/org/openmrs/module/facilitydata/model/CodedFacilityDataQuestionType.java

```diff
@@ -85,4 +90,9 @@ public FacilityDataCodedOption getOptionById(Integer id) {
 		}
 		return ret;
 	}
+
+	public void addFacilityDataCodedOption(FacilityDataCodedOption child) {
+		//child.setQuestionType(this);

```

Comment:
Commented out code is better removed https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Donotcommentoutcode

Reply:
ok